doc/fuzzing.md

```diff
@@ -14,7 +14,7 @@ $ cmake -B build_fuzz \
    -DSANITIZERS=undefined,address,fuzzer
 # macOS users: If you have problem with this step then make sure to read "macOS hints for
 # libFuzzer" on https://github.com/bitcoin/bitcoin/blob/master/doc/fuzzing.md#macos-hints-for-libfuzzer
-$ cmake --build build_fuzz -j$(nproc)
```

Comment:
it seems to me we should likely update https://github.com/bitcoin/bitcoin/blob/19f0dd02c1373dd9d2241ce25cbbcb6831763725/doc/developer-notes.md?plain=1#L224 and https://github.com/bitcoin/bitcoin/blob/14093d5d243f6eb9cfef721c80f92848d95032ee/doc/productivity.md?plain=1#L63-L69 as well (both to cmake and the nproc reference)